### ISSUE TITLE
Update active pseudo-class on sign in link

### ISSUE THREAD

**Issue by darbyemadewell** [department-of-veterans-affairs/va.gov-team#137987]
## Severity

**Approximate Section 508 Severity**: Medium

**VA Severity**: a11y-defect-1

**Rationale**: Lowering the priority on this issue because I've only been able to trigger it by directly applying the :active pseudo-class using Chrome dev tools.

## Behavior
**Expected**: 
The "Sign in" link should have a contrast ratio of at least 4.5:1 in all states.

**Actual**: 
The "Sign in" link has a color contrast of 1.1:1 when the :active pseudo-class is directly applied.

## Examples

<img width="360" height="221" alt="Image" src="https://github.com/user-attachments/assets/d34e7429-5fe8-4594-b308-6c3965fe22c4" />
